cider/tests/functionality/CiderGroupByTest.cpp

```diff
@@ -20,7 +20,7 @@
  */
 
 #include <gtest/gtest.h>
-#include "tests/utils/CiderTestBase.h"
+#include "CiderTestBase.h"
```

Comment:
as a practice, better to include the full file path, see [Names and Order of Includes](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)

Reply:
let's add `include_order` check in `cpplint` later.